webapp/src/main/resources/db/migration/V64__third_party_sync_file_checksum.sql

```diff
@@ -0,0 +1,4 @@
+create table third_party_sync_file_checksum (repository_id bigint not null, locale_id bigint not null, checksum char(32) not null, file_name varchar(255) not null);
+alter table third_party_sync_file_checksum add constraint FK__THIRD_PARTY_CHECKSUM__REPO__ID foreign key (repository_id) references repository (id);
+alter table third_party_sync_file_checksum add constraint FK__THIRD_PARTY_CHECKSUM__LOCALE__ID foreign key (locale_id) references locale (id);
+create index I__THIRD_PARTY__CHECKSUM__REPOSITORY_ID__FILE_NAME__LOCALE_ID on third_party_sync_file_checksum(repository_id, locale_id, file_name);
```

Comment:
We should also have a Unique constraint on this table to make sure that for a file name for a locale associated with a specific repo - we can only ever store the **_last_** checksum - vs. it being possible to store multiple variants over time.

Reply:
That's currently the case with the embedded composite id, as no entry can have an identical repo, locale & file name combination. It will overwrite the last entry with the same combination rather than create one a new one. 

I'll need to add this to the table setup when I update to an auto incrementing id though.